drivers/hmis/app/graphql/mutations/update_external_form_submission.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class UpdateExternalFormSubmission < CleanBaseMutation
+    argument :input, Types::HmisSchema::ExternalFormSubmissionInput, required: true
+    argument :id, ID, required: true
+
+    field :external_form_submission, Types::HmisSchema::ExternalFormSubmission, null: false
+    field :errors, [Types::HmisSchema::ValidationError], null: false, resolver: Resolvers::ValidationErrors
+
+    def resolve(id:, input:)
+      record = Hmis::HmisSchema::ExternalFormSubmission.find(id)
```

Comment:
I think we need to resolve the model here and not the hmis schema object,
```suggestion
      record = HmisExternalApis::ExternalForms::FormSubmission.find(id)
```

Reply:
Yes! I didn't get around to actually exercising this mutation. Thanks for the fix!